src/lib.rs

```diff
@@ -1060,7 +1105,7 @@ mod tests {
   #[test]
   fn bash_slashmatch() {
     // assert!(!glob_match("f[^eiu][^eiu][^eiu][^eiu][^eiu]r", "foo/bar"));
-    assert!(glob_match("foo[/]bar", "foo/bar"));
+    assert!(!glob_match("foo[/]bar", "foo/bar"));
```

Comment:
I don't think we should change these tests either. The previous behavior matches micromatch.

Reply:
I got what you mean, thanks.

Reply:
But I still have two unclear points:
1. As you mentioned in `README.md`, `*` matches zero or more characters, except for path separators (e.g. /). 
So `foo[/]bar` shouldn't match `foo/bar`.
2. In `micromatch`, `a/*/b` matches `a//b` while `a/*` doesn't match `a/` which clearly goes against the meaning of `*`.

Because the above two points have troubled me, I have had to modify the test cases to ensure consistency in their behavior. Can you give me some advice? @devongovett

Reply:
But I still have two unclear points:
1. As you mentioned in `README.md`, `[]` matches one of the characters contained in the brackets, should `path separators` be included?
2. In `micromatch`, `a/*/b` matches `a//b` while `a/*` doesn't match `a/` which clearly goes against the meaning of `*`.

Because the above two points have troubled me, I have had to modify the test cases to ensure consistency in their behavior. Can you give me some advice? @devongovett

Reply:
Can you also give your opinion? @SyMind